source/services/encryption/EncryptionHelper.ts

```diff
@@ -1,14 +1,30 @@
 import { AnsweredForm } from "../../types/Case";
+import {
+  EncryptionExceptionConstructor,
+  EncryptionExceptionInterface,
+  EncryptionExceptionStatus,
+} from "../../types/Encryption";
 import StorageService from "../StorageService";
 
 export interface UserInterface {
   personalNumber: string;
 }
 
-export function EncryptionException(message: string) {
-  this.message = message;
-  this.name = "EncryptionException";
-}
+export const EncryptionException: EncryptionExceptionConstructor = class EncryptionException
```

Comment:
After looking at the code i'm curious to why we call this an Exception? Since it's an extension of the default Error class would not EncryptionError make more sense? Just a thought :)

Or are there something else this class does besides extending the Error class with some additional attributes (name , status) ?

Reply:
Yeah maybe, but technically I think this is a an exception, and the main difference (from my understanding) is that an Exception is just that, used to catch and handle expected edge-cases, where as an Error is truly something that should never be handled. It's just purely semantics though 🙂.

Paraphrasing from [this](https://stackoverflow.com/questions/16142583/whats-the-difference-between-error-and-exception-in-javascript) but:

> JavaScript throws Errors
Developers throws Exceptions